tests/common/test_cgroupapi.py

```diff
@@ -222,6 +223,12 @@ def test_cleanup_old_cgroups_should_move_daemon_pid_on_all_controllers(self):
         self.assertFalse(os.path.exists(old_cpu_cgroup))
         self.assertFalse(os.path.exists(old_memory_cgroup))
 
+        # Assert the event parameters that were sent out
+        _, kwargs = mock_add_event.call_args_list[0]
+        self.assertEquals(kwargs['op'], 'CGroupsCleanUp')
+        self.assertEquals(kwargs['is_success'], True)
+        self.assertEquals(kwargs['message'], 'Successfully cleaned up old cgroups in WALinuxAgent/WALinuxAgent.')
+
```

Comment:
can you also test the aggregation of errors?

Reply:
Updated